client/my-sites/marketing/connections/mailchimp-settings.jsx

```diff
@@ -95,12 +97,12 @@ const MailchimpSettings = ( {
 		<div>
 			<QueryMailchimpLists siteId={ siteId } />
 			<QueryMailchimpSettings siteId={ siteId } />
-			<p>{ translate( 'What Mailchimp list should subscribers be added to?' ) }</p>
+			<p>{ translate( 'What Mailchimp audience should subscribers be added to?' ) }</p>
```

Comment:
:ok: This change will be queued for retranslation. We'll use the existing translations in the meantime.